docs/HARDWARE-VERIFICATION.md

```diff
@@ -0,0 +1,72 @@
+# Verifying EVE on edge nodes
+
+This is a specialized version of EVE-OS installer that installs EVE-OS on the
+underlying edge node and verifies its compatibility. It checks if the necessary
+drivers are present, tests a static and dynamic networking configurations,
+tests the available storage devices, and tests if EVE-OS can spawn a guest
+successfully. Verification is currently supported on x86 edge nodes.
+
+## Running the verification image from boot media
+
+Similar to EVE-OS installer image (see [deployment](DEPLOYMENT.md)), we:
+
+1. Produce a disk-based installer image (e.g., by running `make verification-raw`, which creates the raw image file for verification, similar to what `make installer-raw` does).
+2. Burn the resulting ```verification.raw``` image file onto a USB stick and inserting it into the edge node.
+3. Have the machine boot from USB from BIOS.
+
+To produce the verification.raw image, we can execute the command ```docker run lfedge/eve-verification verification_raw > verification.raw```
+
+In cases such as using HPE iLO or Dell DRAC your BIOS will not be able
+to boot from a disk-based image, so we need an ISO image. The only
+difference is in step #1 that then becomes ```docker run lfedge/eve-verification verification_iso > verification.iso```.
+
+## Running the verification image via iPXE
+
+[iPXE](https://en.wikipedia.org/wiki/IPXE) is a modern Preboot eXecution Environment and
+a boot loader that allows operating system images to be downloaded right at the moment
+of booting (checkout [deployment](DEPLOYMENT.md)).
+To get the necessary images required by iPXE, we can execute the command
+```docker run lfedge/eve-verification verification_net | tar xf -```. Apart from the image, iPXE
```

Comment:
This doesn't work even with 9.12.0-kvm-amd64. The lfedge/eve-verification does not exist and the verification_net argument does not exist.

Reply:
You are right. I have not created the PR for that yet. I will remove it for now and add it later.

Reply:
Probably makes sense to do that PR first, or do it as part of this PR. Without it this documentation PR can not document anything folks can use.